### PR TITLE
ROX-23267: Standard resource config in Scanner V4 install tests 

### PR DESCRIPTION
**Pull request by mclasmeier**
## Description

This PR is *not* anymore about increasing a wait timeout.
Instead we remove our custom (extremely underprovisioned) resource configuration and see how it goes.

## Checklist
- [x] Investigated and inspected CI test results
- [ ] ~~Unit test and regression tests added~~
- [ ] ~~Evaluated and added CHANGELOG entry if required~~
- [ ] ~~Determined and documented upgrade steps~~
- [ ] ~~Documented user facing changes~~

## Testing Performed

Just CI.

### Reminder for reviewers

Could well happen that this turns out to be somewhat flaky, in that case the plan is to pick a larger machine size.

### CODE REVIEW

File: deploy/common/k8sbased.sh [hunk 1]
```diff
@@ -545,9 +545,12 @@ function launch_central {
     # On some systems there's a race condition when port-forward connects to central but its pod then gets deleted due
     # to ongoing modifications to the central deployment. This port-forward dies and the script hangs "Waiting for
     # Central to respond" until it times out. Waiting for rollout status should help not get into such situation.
-    rollout_wait_timeout="4m"
-    if [[ "${IS_RACE_BUILD:-}" == "true" ]]; then
-      rollout_wait_timeout="9m"
+    local rollout_wait_timeout=${CENTRAL_ROLLOUT_WAIT_TIMEOUT:-}
+    if [[ -z "${central_rollout_timeout}" ]]; then
```
Comment (kovayur) [inline deploy/common/k8sbased.sh:549]:
```suggestion
    if [[ -z "${rollout_wait_timeout}" ]]; then
```

**Review by porridge — COMMENTED** [review]
I think I'd prefer to bump the common timeout from 4 to 10 minutes. Bumping it just for scanner-v4 tests:
- introduces more complexity
- does not special-case the race build (not an issue now but another caveat if we ever want to run race builds in scanner v4 tests)
- does not benefit other tests which possibly also suffer from the same issue

**Review by porridge — APPROVED** [review]
🤞🏻

File: deploy/common/k8sbased.sh (hunk 1 above)
Comment (mclasmeier) [reply deploy/common/k8sbased.sh]: Outdated, PR changed completely, hence resolving.

**Pull request merged**